resources/shaders/normalDepthMap.frag

```diff
@@ -12,8 +12,8 @@ void main() {
     vec4 tempInfo = vec4(0, 0, 0, 0);
 
     vec3 normPosition = normalize(-pos);
-    float angleX = abs(normPosition.x / normPosition.z) / limitHorizontalAngle;
-    float angleY = abs(normPosition.y / normPosition.z) / limitVerticalAngle;
+    float angleX = abs(atan(normPosition.x/normPosition.z)) / limitHorizontalAngle;
```

Comment:
It doesn't matter in the long run since the gpu simulation does not use the calculated angles, but it's best to use atan(z, x) instead of atan(x/z).